ambuda/utils/cheda.py

```diff
@@ -18,56 +18,56 @@ class Token:
 
 
 POS = {
-    "n": "noun",
-    "a": "adjective",
-    "v": "verb",
-    "i": "indeclinable",
-    "va": "participle",
-    "vi": "verbal indeclinable",
+    "n": {"en": "noun", "sa": "subanta"},
```

Comment:
Rather than `data[code][language]`, I think `data[language][code]` -- or even `data[code]` where `data` is defined in `[language].py` -- will be easier to maintain, especially once we start supporting dozens of languages.